src/Components/Home.js

```diff
@@ -0,0 +1,68 @@
+import { useEffect } from 'react';
+import { Button, Card, Container } from 'react-bootstrap';
+import { useDispatch, useSelector } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { NavLink } from 'react-router-dom';
+import { fetchCountries, filteredCountries } from '../Redux/reducers/countries';
+
+const Home = () => {
```

Comment:
- [ ] We are expected to follow the design guidelines for this project including typography, and layout. Once more try to check out the screenshots below and make the changes.

| Yours | Guidelines |
| --- | --- |
| ![Screenshot from 2022-12-02 19-24-28](https://user-images.githubusercontent.com/63087491/205361499-ecf1c2e0-1446-4845-9d66-12eed4eb0e2c.png) | ![Screenshot from 2022-12-02 19-27-09](https://user-images.githubusercontent.com/63087491/205361582-9ef96d0e-f5fe-4e80-a3f4-81f168e07813.png) |

- Kindly note that having a nice design increases your chances for a job as recruiters will be checking your app.

Reply:
Hi @FrederickMih, thank you for pointing that out. However, the theme designs for both apps are very much different. I have tried altering my design to match the given design. I hope it resembles better now.